src/electron/routing_service.ts

```diff
@@ -68,28 +80,42 @@ interface NetError extends Error {
 }
 
 // Abstracts IPC with OutlineService in order to configure routing.
-export class WindowsRoutingService implements RoutingService {
+export class RoutingService implements RoutingServiceInterface {
```

Comment:
Looks like the difference between Windows and Linux is much less than we anticipated - nice! I think we can kill `RoutingServiceInterface` - no need to extract interfaces unless clearly needed.